src/services/admin/mentor.service.ts

```diff
@@ -36,3 +49,85 @@ export const updateMentorStatus = async (
     throw new Error('Error updating the mentor status')
   }
 }
+
+export const getAllMentors = async (
+  status: ApplicationStatus
+): Promise<{
+  statusCode: number
+  mentors?: AllMentorsArray[]
```

Comment:
We do have the Mentor interface. Use that.
```suggestion
  mentors?: Mentor[]
```